flang/include/flang/Optimizer/Transforms/Passes.td

```diff
@@ -24,6 +24,14 @@ def AffineDialectPromotion : FunctionPass<"promote-to-affine"> {
   let constructor = "fir::createPromoteToAffinePass()";
 }
 
+def AffineDialectDemotion : FunctionPass<"demote-to-affine"> {
+  let summary = "Converts affine.load and affine.store back to fir operations";
+  let description = [{
+    TODO
```

Comment:
You'll probably want to write something for the descriptions here while it is fresh in mind.

Reply:
Got it, will do for promotion as well.